test/WebJobs.Script.Tests/FunctionMetadataManagerTests.cs

```diff
@@ -6,6 +6,7 @@
 using System.Collections.Immutable;
 using System.Collections.ObjectModel;
 using System.IO;
+using Microsoft.Azure.WebJobs.Script.Abstractions;
```

Comment:
We have E2E tests for proxies (ProxyEndToEndTests) that verify that ListFunctions etc. surface proxy "virtual" functions. We should have an E2E test suite for codeless as well that tests the whole scenario/pipeline there as well.

Reply:
Good point, thanks!
I will add the recommended tests, once we are good with the design, just to avoid throw-away tests, especially if the interface changes a lot. I will leave this unresolved for now.